rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -131,7 +184,56 @@ pub fn tags_to_lanes(tags: &Tags, locale: &Locale, config: &TagsToLanesConfig) -
     Ok(lanes)
 }
 
-fn driving_lane_directions(tags: &Tags, _locale: &Locale, oneway: bool) -> (usize, usize) {
+fn initial_forward_backward(
+    tags: &Tags,
+    locale: &Locale,
+    oneway: Oneway,
+) -> (Vec<LaneBuilder>, Vec<LaneBuilder>) {
+    let (num_driving_fwd, num_driving_back) = driving_lane_directions(tags, locale, oneway);
+
+    let designated = if tags.is("access", "no")
+        && (tags.is("bus", "yes") || tags.is("psv", "yes")) // West Seattle
```

Comment:
Feel free to drop my overly conversational comments at any point ><

Reply:
I will, when I do more refactoring than just moving code around. :)